rmgpy/tools/loader.py

```diff
@@ -39,19 +39,23 @@
 from rmgpy.solver.liquid import LiquidReactor
 from rmgpy.solver.base import TerminationConversion
 
-def loadRMGJob(inputFile, chemkinFile=None, speciesDict=None, generateImages=True, useJava=False, useChemkinNames=False):
+def loadRMGJob(inputFile, chemkinFile=None, speciesDict=None, generateImages=True, useJava=False,
+               useChemkinNames=False, checkDuplicates=True):
 
     if useJava:
         # The argument is an RMG-Java input file
-        rmg = loadRMGJavaJob(inputFile, chemkinFile, speciesDict, generateImages, useChemkinNames=useChemkinNames)
+        rmg = loadRMGJavaJob(inputFile, chemkinFile, speciesDict, generateImages,
```

Comment:
Before I begin the rest of my review, Codacy is flagging this line. One thing I noticed is that `generateImages` is being passed here, but this variable is not defined in the function definition in line 129. I think this argument needs to be removed, unless I am missing something here.

Reply:
You're absolutely right, not sure how that got through previous reviews. Instead of removing the argument, I just fixed the function. I also went ahead and rebased on master.